### PR TITLE
chore: Fix typos in onErrorComplete.md

### PR DESCRIPTION
**Pull request by He-Pin**
Fix typo.

### CODE REVIEW

File: akka-docs/src/main/paradox/stream/operators/Source-or-Flow/onErrorComplete.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # onErrorComplete
 
-Allows completing the stream when a upstream error occur.
+Allows completing the stream when an upstream error occur.
```
Comment (pvlugter) [inline akka-docs/src/main/paradox/stream/operators/Source-or-Flow/onErrorComplete.md:3]:
```suggestion
Allows completing the stream when an upstream error occurs.
```

File: akka-docs/src/main/paradox/stream/operators/index.md [hunk 2]
```diff
@@ -362,7 +362,7 @@ For more background see the @ref[Error Handling in Streams](../stream-error.md)
 | |Operator|Description|
 |--|--|--|
 |Source/Flow|<a name="maperror"></a>@ref[mapError](Source-or-Flow/mapError.md)|While similar to `recover` this operators can be used to transform an error signal to a different one *without* logging it as an error in the process.|
-|Source/Flow|<a name="onerrorcomplete"></a>@ref[onErrorComplete](Source-or-Flow/onErrorComplete.md)|Allows completing the stream when a upstream error occur.|
+|Source/Flow|<a name="onerrorcomplete"></a>@ref[onErrorComplete](Source-or-Flow/onErrorComplete.md)|Allows completing the stream when an upstream error occur.|
```
Comment (pvlugter) [inline akka-docs/src/main/paradox/stream/operators/index.md:365]:
```suggestion
|Source/Flow|<a name="onerrorcomplete"></a>@ref[onErrorComplete](Source-or-Flow/onErrorComplete.md)|Allows completing the stream when an upstream error occurs.|
```

**Review by pvlugter — APPROVED** [review]
LGTM. Also `occur` -> `occurs` while we're here.

**Review by johanandren — APPROVED** [review]
Thanks

**Pull request merged**